include/render.h

```diff
@@ -23,6 +23,7 @@
 #include <string>
```

Comment:
Please add `#include <cstring>`, as my compiler (GCC 13.1.0, Gentoo Linux) complains about `std::memcpy` not being a known symbol.

Reply:
Done.